draft-ietf-quic-recovery.md

```diff
@@ -379,8 +387,9 @@ default values.
 On subsequent RTT samples, smoothed_rtt and rttvar evolve as follows:
 
 ~~~
-ack_delay = acknowledgement delay from ACK frame
-ack_delay = min(ack_delay, max_ack_delay)
+ack_delay = decoded ACK Delay field from ACK frame
```

Comment:
I think the old text was clearer in this case.
```suggestion
ack_delay = acknowledgement delay from ACK frame
```

Reply:
it's setting a variable, so spelling out that it is a decoded field seems more explicit.

Reply:
I think the most correct clear option is "decoded acknowledgement delay from the ACK Delay field in the ACK frame", but that's quite long.

Reply:
Maybe something like `ack_frame.ack_delay`?